### PR TITLE
tests: Add the description for linux bridge integration tests

### PR DESCRIPTION
**Pull request by liangwen12year**

### CODE REVIEW

**Comment by liangwen12year** [comment]
The following are the generated yaml states:

```
- question: Configure eth2 with the controller linux-br0
  answer: |
    ---
    interfaces:
    - name: eth2
      type: ethernet
      state: up
      controller: linux-br0
- question: Create the linux bridge linux-br0 with the ethernet port eth1 and eth2. For both ethernet port eth1 and eth2, configure the stp hairpin mode to false, configure the stp path cost to 100, configure the stp priority to 32
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port:
        - name: eth1
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
        - name: eth2
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
- question: Create the bond interface testbond0 with the controller {}, the bonding mode balance-rr, and ethernet port eth1 linux-br0
  answer: |
    ---
    interfaces:
    - name: testbond0
      type: bond
      state: up
      controller: linux-br0
      link-aggregation:
        mode: balance-rr
        port:
        - eth1
- question: Configure the linux bridge linux-br0 with the group address 01:80:C2:00:00:04
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          group-addr: 01:80:C2:00:00:04
- question: Configure the linux bridge linux-br0 with multicast-snooping set to true
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          multicast-snooping: true
- question: Bring up ethernet device eth1 with empty controller
  answer: |
    ---
    interfaces:
    - name: eth1
      type: ethernet
      state: up
      controller: ''
- question: bring up the linux bridge linux-br0
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
- question: Configure the linux bridge linux-br0 with port eth2 attached, set the stp hairpin mode to false, set the stp path cost to 100, set the stp priority to 32
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port:
        - name: eth2
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
- question: Create bridge linux-br0, add the ethernet port eth1 and eth2 to the linux bridge. In ethernet port eth1, set the stp hairpin mode to false, stp path cost to 100, stp priority to 32.
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port:
        - name: eth1
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
        - name: eth2
- question: Set accepting all mac to true for linux bridge linux-br0
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      accept-all-mac-addresses: true
- question: Set accepting all mac to false for linux bridge linux-br0
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      accept-all-mac-addresses: false
- question: Configure the linux bridge linux-br0 with the multi-cast router disabled
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          multicast-router: disabled
- question: Configure the linux bridge linux-br0 with the multi-cast router 0
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          multicast-router: disabled
- question: Configure the linux bridge linux-br0 with the multi-cast router enabled
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          multicast-router: enabled
- question: Configure the linux bridge linux-br0 with the multi-cast router 2
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          multicast-router: enabled
- question: Create the linux bridge linux-br0 with vlan default pvid set to 5
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          vlan-default-pvid: 5
- question: Create the linux-bridge linux-br0 with the IPv4 disabled and IPv6 disabled, and the bridge port testbond0 attached. Create the bond testbond0 with the controller linux-br0, the bonding mode balance-rr and the bonding port eth1
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      ipv4:
        enabled: false
      ipv6:
        enabled: false
      bridge:
        options:
          group-fwd-mask: 0
          mac-ageing-time: 300
          multicast-snooping: true
          stp:
            enabled: true
            forward-delay: 15
            hello-time: 2
            max-age: 20
            priority: 32768
        port:
        - name: testbond0
          stp-hairpin-mode: false
          stp-path-cost: 100
          stp-priority: 32
    - name: testbond0
      type: bond
      state: up
      controller: linux-br0
      link-aggregation:
        mode: balance-rr
        port:
        - eth1
- question: Create the bridge linux-br0 with the ethernet port eth1
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port:
        - name: eth1
- question: Create the linux bridge linux-br0 with the vlan protocol 802.1ad
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        options:
          vlan-protocol: 802.1ad
- question: Bring up the linux bridge linux-br0 with the empty port, configure the bonding interface test-bond0 with the bonding mode balance-rr and ethernet port eth1
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port: []
    - name: test-bond0
      type: bond
      state: up
      link-aggregation:
        mode: balance-rr
        port:
        - eth1
- question: Remove the bridge linux-br1
  answer: |
    ---
    interfaces:
    - name: linux-br1
      type: linux-bridge
      state: absent
- question: Set the description 'secondary' for the eth1 interface, set the description 'primary' for the eth2 interface
  answer: |
    ---
    interfaces:
    - name: eth1
      description: secondary
      type: ethernet
      state: up
    - name: eth2
      description: primary
      type: ethernet
      state: up
- question: Delete bridge linux-br0
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: absent
- question: Create the linux bridge linux-br0 with the MAC address 36:30:DB:60:75:A0, DHCP4 enabled, bridge stp option disabled, port eth1 and eth2 attached
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      mac-address: 5A:1C:E3:68:EE:84
      ipv4:
        enabled: true
        dhcp: true
        auto-dns: true
        auto-gateway: true
        auto-routes: true
      bridge:
        options:
          stp:
            enabled: false
        port:
        - name: eth1
        - name: eth2
- question: Remove the linux bridge linux-br0 and keep the ethernet port eth1 up
  answer: |
    ---
    interfaces:
    - name: eth1
      type: ethernet
      state: up
      controller: ''
    - name: linux-br0
      type: linux-bridge
      state: absent
- question: Create the linux bridge linux-br0 with the ethernet port eth2 attached
  answer: |
    ---
    interfaces:
    - name: linux-br0
      type: linux-bridge
      state: up
      bridge:
        port:
        - name: eth2
```

**Comment by liangwen12year** [comment]
```
description: Configure eth2 with the controller linux-br0
interfaces:
- name: eth2
  type: ethernet
  state: up
  controller: linux-br0
description: Create the linux bridge linux-br0 with the ethernet port eth1 and eth2. For both ethernet port eth1 and eth2, do not allow traffic to be sent back to the same port on which it was received, configure the stp path cost to 100, configure the stp priority to 32
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
    - name: eth2
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
description: Create the bond interface testbond0 with the controller {}, the bonding mode balance-rr, and ethernet port eth1 linux-br0
interfaces:
- name: testbond0
  type: bond
  state: up
  controller: linux-br0
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Configure the linux bridge linux-br0 with the group address 01:80:C2:00:00:04
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      group-addr: 01:80:C2:00:00:04
description: Configure the linux bridge linux-br0 with multicast-snooping set to true
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-snooping: true
description: Bring up ethernet device eth1 with empty controller
interfaces:
- name: eth1
  type: ethernet
  state: up
  controller: ''
description: bring up the linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
description: Configure the linux bridge linux-br0 with port eth2 attached, do not allow traffic to be sent back to the same port on which it was received, set the stp path cost to 100, set the stp priority to 32
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth2
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
description: Create bridge linux-br0, add the ethernet port eth1 and eth2 to the linux bridge. In ethernet port eth1, do not allow traffic to be sent back to the same port on which it was received, set stp path cost to 100, set stp priority to 32.
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
    - name: eth2
description: Set accepting all mac to true for linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  accept-all-mac-addresses: true
description: Set accepting all mac to false for linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  accept-all-mac-addresses: false
description: Configure the linux bridge linux-br0 with the multi-cast router auto
interfaces: []
description: Configure the linux bridge linux-br0 with the multi-cast router 1
interfaces: []
description: Configure the linux bridge linux-br0 with the multi-cast router disabled
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: disabled
description: Configure the linux bridge linux-br0 with the multi-cast router 0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: disabled
description: Configure the linux bridge linux-br0 with the multi-cast router enabled
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: enabled
description: Configure the linux bridge linux-br0 with the multi-cast router 2
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: enabled
description: Create the linux bridge linux-br0 with vlan default pvid set to 5
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      vlan-default-pvid: 5
description: Create the linux-bridge linux-br0 with the IPv4 disabled and IPv6 disabled, and the bridge port testbond0 attached. Create the bond testbond0 with the controller linux-br0, the bonding mode balance-rr and the bonding port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  ipv4:
    enabled: false
  ipv6:
    enabled: false
  bridge:
    options:
      group-fwd-mask: 0
      mac-ageing-time: 300
      multicast-snooping: true
      stp:
        enabled: true
        forward-delay: 15
        hello-time: 2
        max-age: 20
        priority: 32768
    port:
    - name: testbond0
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
- name: testbond0
  type: bond
  state: up
  controller: linux-br0
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Create the bridge linux-br0 with the ethernet port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
description: Create the linux bridge linux-br0 with the vlan protocol 802.1ad
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      vlan-protocol: 802.1ad
description: Create the linux bridge linux-br0 with the vlan protocol 802.1ad
interfaces: []
description: Bring up the linux bridge linux-br0 with the empty port, configure the bonding interface test-bond0 with the bonding mode balance-rr and ethernet port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port: []
- name: test-bond0
  type: bond
  state: up
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Remove the bridge linux-br1
interfaces:
- name: linux-br1
  type: linux-bridge
  state: absent
description: Create the linux bridge linux-br0 with the MAC address 36:30:DB:60:75:A0, DHCP4 enabled, bridge stp option disabled, port eth1 and eth2 attached
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  mac-address: 12:45:03:76:AA:B7
  ipv4:
    enabled: true
    dhcp: true
    auto-dns: true
    auto-gateway: true
    auto-routes: true
  bridge:
    options:
      stp:
        enabled: false
    port:
    - name: eth1
    - name: eth2
description: Delete bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: absent
description: Set the description 'secondary' for the eth1 interface, set the description 'primary' for the eth2 interface
interfaces:
- name: eth1
  description: secondary
  type: ethernet
  state: up
- name: eth2
  description: primary
  type: ethernet
  state: up
description: Remove the linux bridge linux-br0 and keep the ethernet port eth1 up
interfaces:
- name: eth1
  type: ethernet
  state: up
  controller: ''
- name: linux-br0
  type: linux-bridge
  state: absent
description: Create the linux bridge linux-br0 with the ethernet port eth2 attached
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth2
```

File: tests/integration/linux_bridge_test.yaml [hunk 1]
```diff
@@ -0,0 +1,248 @@
+description: Configure eth2 with the controller linux-br0
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: Attach eth2 to controller linux-br0

File: tests/integration/linux_bridge_test.yaml [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+    port:
+    - name: eth1
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+    - name: eth2
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+description: Create the bond interface testbond0 with the controller linux-br0, the bonding mode balance-rr, and ethernet port eth1
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: Create bond interface testbond0 with eth1 as port and then attach to controller linux-br0

File: tests/integration/linux_bridge_test.yaml [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+      group-addr: 01:80:C2:00:00:04
+description: Configure the linux bridge linux-br0 with multicast-snooping set to true
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-snooping: true
+description: Set the controller of ethernet device eth1 to empty
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: Detach eth1 from any controller

File: tests/integration/linux_bridge_test.yaml [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+- name: eth1
+  type: ethernet
+  state: up
+  controller: ''
+description: bring up the linux bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+description: Configure the linux bridge linux-br0 with port eth2 attached, do not allow traffic to be sent back to the same port on which it was received, set the stp path cost to 100, set the stp priority to 32
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: We already explained the hairpin-mode in previous test, I think it is better just say `disable stp hairpin-mode` here as diversity.

File: tests/integration/linux_bridge_test.yaml [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+      multicast-router: disabled
+description: Configure the linux bridge linux-br0 with the multi-cast router enabled
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-router: enabled
+description: Configure the linux bridge linux-br0 with the multi-cast router 2
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: Even we desired number 2, but when generating diff, nmstate uses `multicast-router: enabled`. So I think you need to change your `apply_with_description()` there.

File: tests/integration/linux_bridge_test.yaml [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+interfaces:
+- name: linux-br1
+  type: linux-bridge
+  state: absent
+description: Delete bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: absent
+description: Create the linux bridge linux-br0 with MAC address, DHCP4 enabled, bridge stp option disabled, port eth1 and eth2 attached
```
Comment (cathay4t) [inline tests/integration/linux_bridge_test.yaml]: with MAC address set to `4A:75:EE:88:87:74`

**Review by cathay4t — CHANGES_REQUESTED** [review]

File: tests/integration/linux_bridge_test.yaml (hunk 2 above)
Comment (liangwen12year) [reply tests/integration/linux_bridge_test.yaml]: It seems that you ignored the bonding mode.

**Comment by liangwen12year** [comment]
@cathay4t , thanks for taking time reviewing it, I addressed all your comments.

**Review by cathay4t — APPROVED** [review]

**Comment by liangwen12year** [comment]
```
description: Attach eth2 to controller linux-br0
interfaces:
- name: eth2
  type: ethernet
  state: up
  controller: linux-br0
description: Create the linux bridge linux-br0 with the ethernet port eth1 and eth2. For both ethernet port eth1 and eth2, do not allow traffic to be sent back to the same port on which it was received, configure the stp path cost to 100, configure the stp priority to 32
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
    - name: eth2
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
description: Create bond interface testbond0 with bonding mode balance-rr and eth1 as port, then attach it to controller linux-br0 eth1
interfaces:
- name: testbond0
  type: bond
  state: up
  controller: linux-br0
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Configure the linux bridge linux-br0 with the group address 01:80:C2:00:00:04
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      group-addr: 01:80:C2:00:00:04
description: Configure the linux bridge linux-br0 with multicast-snooping set to true
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-snooping: true
description: Detach eth1 from any controller
interfaces:
- name: eth1
  type: ethernet
  state: up
  controller: ''
description: bring up the linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
description: Configure the linux bridge linux-br0 with port eth2 attached, disable stp hairpin-mode, set the stp path cost to 100, set the stp priority to 32
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth2
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
description: Create bridge linux-br0, add the ethernet port eth1 and eth2 to the linux bridge. In ethernet port eth1, do not allow traffic to be sent back to the same port on which it was received, set stp path cost to 100, set stp priority to 32.
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
    - name: eth2
description: Set accepting all mac to true for linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  accept-all-mac-addresses: true
description: Set accepting all mac to false for linux bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  accept-all-mac-addresses: false
description: Configure the linux bridge linux-br0 with the disabled multi-cast router
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: disabled
description: Configure the linux bridge linux-br0 with the disabled multi-cast router
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: disabled
description: Configure the linux bridge linux-br0 with the enabled multi-cast router
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: enabled
description: Configure the linux bridge linux-br0 with the enabled multi-cast router
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      multicast-router: enabled
description: Create the linux bridge linux-br0 with vlan default pvid set to 5
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      vlan-default-pvid: 5
description: Create the linux-bridge linux-br0 with the IPv4 disabled and IPv6 disabled, set group-forward-mask to 0, set mac-ageing-time to 300 seconds, enable multicast-snooping, activate Spanning Tree Protocol (STP) with a forward-delay of 15 seconds, hello-time of 2 seconds, max-age of 20 seconds, and a priority of 32768. The bridge port testbond0 attached with stp-hairpin-mode set to false, stp-path-cost set to 100, stp-priority set to 32. Create the bond testbond0 with the controller linux-br0, the bonding mode balance-rr and the bonding port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  ipv4:
    enabled: false
  ipv6:
    enabled: false
  bridge:
    options:
      group-forward-mask: 0
      mac-ageing-time: 300
      multicast-snooping: true
      stp:
        enabled: true
        forward-delay: 15
        hello-time: 2
        max-age: 20
        priority: 32768
    port:
    - name: testbond0
      stp-hairpin-mode: false
      stp-path-cost: 100
      stp-priority: 32
- name: testbond0
  type: bond
  state: up
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Create the bridge linux-br0 with the ethernet port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth1
description: Create the linux bridge linux-br0 with the vlan protocol 802.1ad
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    options:
      vlan-protocol: 802.1ad
description: Bring up the linux bridge linux-br0 with the empty port, configure the bonding interface test-bond0 with the bonding mode balance-rr and ethernet port eth1
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port: []
- name: test-bond0
  type: bond
  state: up
  link-aggregation:
    mode: balance-rr
    port:
    - eth1
description: Remove the bridge linux-br1
interfaces:
- name: linux-br1
  type: linux-bridge
  state: absent
description: Delete bridge linux-br0
interfaces:
- name: linux-br0
  type: linux-bridge
  state: absent
description: Set the description 'secondary' for the eth1 interface, set the description 'primary' for the eth2 interface
interfaces:
- name: eth1
  description: secondary
  type: ethernet
  state: up
- name: eth2
  description: primary
  type: ethernet
  state: up
description: Create the linux bridge linux-br0 with MAC address C6:1D:87:D3:06:62, DHCP4 enabled, bridge stp option disabled, port eth1 and eth2 attached
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  mac-address: C6:1D:87:D3:06:62
  ipv4:
    enabled: true
    dhcp: true
  bridge:
    options:
      stp:
        enabled: false
    port:
    - name: eth2
    - name: eth1
description: Remove the linux bridge linux-br0 and keep the ethernet device eth1 up
interfaces:
- name: eth1
  type: ethernet
  state: up
  controller: ''
- name: linux-br0
  type: linux-bridge
  state: absent
description: Create the linux bridge linux-br0 with the ethernet port eth2 attached
interfaces:
- name: linux-br0
  type: linux-bridge
  state: up
  bridge:
    port:
    - name: eth2
```

**Pull request merged**